src/store.ts

```diff
@@ -1,10 +1,11 @@
 import { applyMiddleware, createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
 import thunk from 'redux-thunk';
 import rootReducer from './reducers';
 
 const store = createStore(
   rootReducer,
-  applyMiddleware(thunk)
+  composeWithDevTools(applyMiddleware(thunk))
```

Comment:
Set `shouldRecord` to false?

From https://medium.com/@zalmoxis/using-redux-devtools-in-production-4c5b56c5600f

>  The RAM consumption is optimized by the limit of actions to be stored, which is `50` by default. You can disable recording any actions by setting shouldRecord option to false (it’s true by default), then it will not store the history till clicking the record button in the monitor window.